### DISCUSSION TITLE
HTML files exported do not display properly in browsers

### DISCUSSION THREAD

**Discussion by helincao** [opengeos/leafmap#39] [Q&A] [answered]
Code snippet to export Map() to a file

```Python
import leafmap

m_leafmap = leafmap.Map()
m_leafmap.save('leafmap_export.html')
```

Error seen in my browser console - Version 91.0.4472.77 (Official Build) (x86_64)
Safari, Version 10.0 (12602.1.50.0.10), has the same issue.

![image](https://user-images.githubusercontent.com/18755763/121965663-382d3100-cd22-11eb-90b7-0c512b7a7d47.png)

**Comment by giswqs** [comment, marked as answer]
This is a known issue with ipyleaflet. See https://github.com/jupyter-widgets/ipyleaflet/issues/841. Try the following code and see if you get the same error. If it is, then it is an issue that should be fixed by ipyleaflet. 

```
import ipyleaflet

m = ipyleaflet.Map()
m.save('mymap.html')
```

**Reply by helincao** [reply]
Yep, using ipyleaflet, sometime the output file works, something not. I guess I will wait for their resolution then. Thanks!

**Comment by giswqs** [comment]
@helincao This issue has been fixed by @sackh. Please update leafmap using `leafmap.update_package()`.

**Comment by helincao** [comment]
It works great! Thanks!
